add cloud cli video and alias info (#4320)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/cloud-cli-installation.md

* [Automatic deferral](/docs/cloud/about-cloud-develop-defer) of build artifacts to your Cloud project's production environment. 
* Speedier, lower-cost builds.
* Support for dbt Mesh ([cross-project `ref`](/docs/collaborate/govern/project-dependencies)),
https://github.com/dbt-labs/docs.getdbt.com/pull/4320* Significant platform improvements, to be released over the coming months.

## Prerequisites 
The dbt Cloud CLI is available in all [deployment regions](/docs/cloud/about-cloud/regions-ip-addresses) and for both multi-tenant and single-tenant accounts (Azure single-tenant not supported at this time).

- Ensure you are using dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
- Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.

> - Avoid using [PrivateLink](/cloud/secure/about-privatelink).

Author: ```suggestion
- Avoid using [PrivateLink](/docs/cloud/secure/about-privatelink).
```

## Install dbt Cloud CLI

> You can install the dbt Cloud CLI on the command line by using one of these methods. For a video instruction step-by-step guide, refer to the [FAQs](#faqs).

Reviewer: ```suggestion
You can install the dbt Cloud CLI on the command line by using one of these methods. For a step-by-step video guide, refer to the [FAQs](#faqs).
```

Before you begin, make sure you have [Homebrew installed](http://brew.sh/) in your code editor or command line terminal. Refer to the [FAQs](#faqs) if your operating system runs into path conflicts.

1. Run the following command to verify that you don't already have dbt Core installed:
   
  ```bash
  which dbt
  ```
   - This should return a `dbt not found`. If the dbt help text appears, use `pip uninstall dbt` to remove dbt Core from your machine. <br />
  
2. Install the dbt Cloud CLI with Homebrew: 

>    - First, remove a tap, which is like a separate repository for packages, from Homebrew. This stops Homebrew from installing packages from that repository:

Reviewer: Adding specificity might help users following the instructions. 
```suggestion
   - First, remove the dbt-labs tap, the separate repository for packages, from Homebrew. This prevents Homebrew from installing packages from that repository:
```

      ```bash
      brew untap dbt-labs/dbt
   -  Then run `brew tap` to add and install the dbt Cloud CLI as a package:
      ```bash
      brew tap dbt-labs/dbt-cli
      ```
   - Lastly, install the dbt Cloud CLI with Homebrew:
      ```bash
      brew install dbt
      ```

> 3. Verify the installation by running `dbt --help` from the command line. 

Reviewer: Should we drop some sample text of what the expected output is?

>    * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a global dbt version installed.

Reviewer: Should we drop some sample text of what the expected output is?

Author: i thought about it and opted to leave it out because `dbt --help` returns a pretty long output (basically it gives users all the flags/commands they can use) so i decided to leave it out and thought that the 'help text...' would be clear enough

Reviewer: "`dbt --help` returns a pretty long output.." is the text that indicates it is successfully running for cloud going to be immediately apparent in this output or will they have to dig for it? Maybe including just a sample of what to look for so they know 100% it was successful?

Reviewer: I think I'm maybe just confused by what this sentence is saying. Are we saying that the ability to run `dbt --help` is enough to confirm installation, or are they supposed to see something that specifically states it's the Cloud CLI and not the Core CLI?  If it's the former, maybe we can clarify it by saying something like: "If the installation was successful, executing `dbt --help` will produce a list of commands available to run."

Author: ok maybe this clarifies it further?

![Screenshot 2023-10-25 at 10 39 43](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/7f3f76e9-5b00-421f-92ae-7394cc78e86f)

   * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.

> 4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.

Reviewer: Might be helpful to give an idea of what `dbt compile` does in the instructions
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

> **Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 

Reviewer: ```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine.  
```

Note that if you are using VS Code, you must restart it to pick up modified environment variables.
:::

3. Verify the installation by running `./dbt --help` from the command line. 
   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
   * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.

> 4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.

Reviewer: Same suggestions as above
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

:::

3. Verify the installation by running `./dbt --help` from the command line. 
   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
   * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.

> 4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.

Reviewer: ```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

> **Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 

Reviewer: ```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine. 
```

  pip3 install dbt-core==VERSION
  ```

> 4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.

Reviewer: ```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

> **Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 

Reviewer: ```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine. 
```

You can always uninstall the dbt Cloud CLI to return to using dbt Core.
</details>

<details>

> <summary>Are there any instructional videos to help guide me through installation? </summary>

Reviewer: I'm not opposed to this being in the FAQs, but given what we know about scroll rates, this might get more views if we embed it directly into the instructions above, rather than/in addition to putting it here.

Author: Yea i also tried adding it in the install header but then it took up too much of the page before introducing the install steps. I could link to it but that also means it'll navigate the user to another tab (and away from the docs). Which is why i figured the faq and a link to the faq was a smoother experience (as opposed to navigating them away from the docs)

Reviewer: I'll ask if we can get pop out windows for videos
